### DISCUSSION TITLE
Updating a modified Local version of TailPress Best Practice?

### DISCUSSION THREAD

**Discussion by moxyinc** [tailpress/tailpress#34] [Q&A] [answered]
I've developed a local theme using TailPress, and I see now that tailwindcss has updated. (I actually tried just installing the new version of tailwindcss via npm last night but in the process my "npm run watch" stopped working, and things basically felt a bit dodgy across the site. I reverted back to the previous tailwind version via npm install and everything was back to normal.) 

I see that tailwind has updated to the new version (which is great.) What's the best workflow to update my local theme, without losing what I've added in the Resources folder, the tailwind.config and tailpress.json files with fonts and color customizations?

**Comment by jeffreyvr** [comment, marked as answer]
You probably encountered this issue: https://github.com/jeffreyvr/tailpress/issues/32

By specifying the PurgeCSS options you can resolve the issue, like is done here: https://github.com/jeffreyvr/tailpress/blob/master/tailwind.config.js#L9

It's not ideal, because all root template files we wan't to watch for now need to be added separately - but as long as there is no other solution we'd probably have to do it this way.

**Reply by moxyinc** [reply]
Thanks, Jeff - yes - that indeed looks like what it was. Thanks for the heads up.
